### PR TITLE
Add docker repo for all sle12 supported docker

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

**Comment by MalloZup** [comment]
i will test the pr asap

**Comment by MalloZup** [comment]
@mbologna feel free to review it. thx

File: salt/minion/testsuite.sls [hunk 1]
```diff
@@ -19,7 +19,7 @@ cucumber_requisites:
     - require:
       - sls: minion.repos
 
-{% if grains['os'] == 'SUSE' and grains['osrelease'] == '12.2' %}
+{% if grains['os'] == 'SUSE' and (grains['osrelease'] == '12.2' or grains['osrelease'] == '12.1' or grains['osrelease'] == '12')  %}
```
Comment (mbologna) [inline salt/minion/testsuite.sls]: This could be simplified:

`{% if '12' in grains['osrelease'] %}`

what do you think?

**Comment by MalloZup** [comment]
ok done

**Comment by moio** [comment]
Thanks!

**Pull request merged**